mmocr/datasets/preparers/obtainers/naive_data_obtainer.py

```diff
@@ -186,8 +186,7 @@ def move(self, mapping: List[Tuple[str, str]]) -> None:
             if '*' in src:
                 mkdir_or_exist(dst)
                 for f in glob.glob(src):
-                    if not osp.exists(
-                            osp.join(dst, osp.relpath(f, self.data_root))):
+                    if not osp.exists(osp.join(dst, osp.basename(f))):
```

Comment:
This modification may break other dataset preparers. Need to make the if condition compatible with both cases.